### PR TITLE
Camera switch env var added

### PR DESCRIPTION
**Pull request by viciopoli01**
## Description

This commit allows preventing the camera node to start when the dt-duckiebot-interface is launched.
The normal setup remains the same, what change is that in the case one wants to disable the camera, for various reasons (adding a new camera that needs more than a simple interface for example), it can start the dt-duckiebot -interface passing the environment variable  DISABLE_CAMERA='on'.

## Type of change

Please mark the relevant options with X:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change is constrained only to one package, any interfaces to nodes from other packages are maintained
- [ ] This change is constrained only to packages in this repository, any interfaces to nodes from other repositories are maintained
- [ ] This change required a documentation update
- [ ] This change is only a documentation update
- [ ] This change restructures the repository (at level higher than package level)
- [ ] This change requires a change in the Duckiebook (if yes, please describe and, if possible, make a pull request for an edit in the book and link it here)

## Tests

Please describe all tests ran and their outcome. It is highly recommended that you at least do the tests listed by default, even if you only changed the documentation.

- [X] Made the Duckiebot move with keyboard control
- [X] Visually checked the published camera stream, when the DISABLE_CAMERA if not set 
- [ ] Verified that the lane following demo works
- [ ] Verified that the indefinite navigation demo works

## Style compliance
Please make sure that all your changes conform to the Duckietown (code style)[link] and code (documentation style)[link] guidelines:

It's only a launch file modification there is no code change.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I checked that this pull request is compliant with the Duckietown code style
- [ ] I checked that this pull request is compliant with the Duckietown code documentation style

## Next steps
Please describe if you have any questions, comments, or doubts towards the Duckietown software guardians regarding implementation, code style, running tests, documentation, etc.

## FOR THE REVIEWERS
__IMPORTANT:__ DO NOT CHANGE THIS SECTION!

- [ ] Functionality (is the change necessary and is it implemented in the most appropriate way): @liampaull or @afdaniele
- [ ] Integration testing (verify the tests and run additional if necessary): @?
- [ ] Code style compliance: @gibernas
- [ ] Documentation compliance: @aleksandarpetrov

_To the reviewers:_ If you deem necessary, request code edits, more tests, documentation update, or additional reviewing. __Do not review your own code!__

_PR Template, last edit 23 Aug 2019_

### CODE REVIEW

**Review by trevphil — APPROVED** [review]

**Comment by afdaniele** [comment]
It looks ok, but change the env variable to `DT_SENSOR_CAMERA_DISABLE` and its possible values to `[1, 0]`, avoid strings like `yes`, `Yes`, `YES`, `on`, `On`, `oFF`, there is too much variability there.
These changes would make this PR more inline with the future direction of the codebase.

**Comment by afdaniele** [comment]
@viciopoli01 ^

**Comment by viciopoli01** [comment]
Tested with other Duckiebots.
Can start and stop the camera node.

File: launchers/default.sh [hunk 1]
```diff
@@ -25,7 +25,8 @@ dt-exec roslaunch --wait \
     duckiebot_interface all_drivers.launch \
     veh:="$VEHICLE_NAME" \
     robot_type:="$ROBOT_TYPE" \
-    robot_configuration:="$ROBOT_CONFIGURATION"
+    robot_configuration:="$ROBOT_CONFIGURATION" \
+    dt_sensor_disable_camera:="$DT_SENSOR_CAMERA_DISABLE"
```
Comment (afdaniele) [inline launchers/default.sh:29]: this should go, the env var will be read by the launchfile directly

File: packages/duckiebot_interface/launch/all_drivers.launch [hunk 2]
```diff
@@ -8,6 +8,9 @@
          doc="The configuration of the robot. For example DB19, DB20, ..."/>
     <arg name="robot_xurdf" value="$(find duckiebot_interface)/urdf/$(arg robot_type).urdf.xacro"
          doc="Location of the URDF XACRO file for the robot type"/>
+    <arg name="dt_sensor_disable_camera" default="0"
+         doc="If 1 the camera node is not started."/>
```
Comment (afdaniele) [inline packages/duckiebot_interface/launch/all_drivers.launch:12]: This should be
`<arg name="camera_disabled" default="$(optenv DT_SENSOR_CAMERA_DISABLE 0)" doc="Disable camera" />`

File: packages/duckiebot_interface/launch/all_drivers.launch [hunk 3]
```diff
@@ -16,11 +19,13 @@
           name="robot_state_publisher"/>
 
     <!-- Camera Node: reads frames from a physical camera and produces ROS messages -->
-    <group if="$(eval arg('robot_type') in ['duckiebot', 'watchtower', 'traffic_light', 'duckiedrone'])">
-        <include file="$(find camera_driver)/launch/camera_node.launch">
-            <arg name="veh" value="$(arg veh)"/>
-            <arg name="param_file_name" value="$(arg robot_type)"/>
-        </include>
+    <group if="$(eval arg('dt_sensor_disable_camera') != 1)">
```
Comment (afdaniele) [inline packages/duckiebot_interface/launch/all_drivers.launch:22]: this should be 
`<group unless="$(arg camera_disabled)">`

**Review by afdaniele — CHANGES_REQUESTED** [review]

File: launchers/default.sh (hunk 1 above)
Comment (viciopoli01) [reply launchers/default.sh]: Done in the new commit. Tested with lab DB21M.

File: packages/duckiebot_interface/launch/all_drivers.launch (hunk 2 above)
Comment (afdaniele) [reply packages/duckiebot_interface/launch/all_drivers.launch]: use exactly what I posted

**Review by afdaniele — APPROVED** [review]

**Pull request merged**